osbs/conf.py

```diff
@@ -286,6 +286,10 @@ def get_source_registry_uri(self):
     def get_pulp_registry(self):
         return self._get_value("pulp_registry_name", self.conf_section, "pulp_registry_name")
 
+    def get_pulp_manifest_lists(self):
+        return self._get_value("pulp_manifest_lists", self.conf_section,
```

Comment:
as a general issue, yes, manifest lists are independent of pulp registry. As a practical matter, if you enable manifest lists in atomic_reactor and your pulp registry doesn't support them, things will break and the point of this entry to is to keep those things from breaking.